__tests__/endpoints.test.js

```diff
@@ -206,3 +207,36 @@ describe("GET /api/articles/:article_id - with comment_count", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("Status 200 - should return an array of article objects containing all the correct properties", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles).toBeInstanceOf(Array);
+        expect(articles).toHaveLength(12);
+        articles.forEach((article) => {
+          expect(article).toMatchObject({
+            article_id: expect.any(Number),
+            title: expect.any(String),
+            author: expect.any(String),
+            created_at: expect.any(String),
+            topic: expect.any(String),
+            votes: expect.any(Number),
+            body: expect.any(String),

```

Comment:
double check which properties are listed on the ticket!

Reply:
Sorted :)